examples/segnet/README.md

```diff
@@ -0,0 +1,39 @@
+SegNet
+======
+
+# Preparation
+
+Please create `class_weight.npy` using calc_weight.py first. Just run:
+
```

Comment:
Could you summarize the difference between the original implementation and the ChainerCV implementation in this README?
Like you did in the PR comment?